io.catenax.repair_certificate/RELEASE_NOTES.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this model will be documented in this file.
+
+## [Unreleased]
+
+## [1.0.0] - 2024-03-04
```

Comment:
here date change required to 11th march for MS3 approval on that day.

Reply:
Updated